images/update-config/update-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -e
+
+set -o pipefail
+
+function update_config () {
+    local config="$1"
+    kubectl create configmap $config --from-file=$config.yaml --dry-run -o yaml | kubectl replace configmap $config -f -
+}
+
+function main () {
+    git clone "https://github.com/e2e-win/e2e-win-prow-deployment"
+    pushd "e2e-win-prow-deployment/prow-cluster/configmaps"
+        update_config "config"
```

Comment:
We should somehow test the config before uploading. Clone test-infra and bazel run . Though we would add the overhead of bazel in the image. Spend some time to see what bazel does when it tests the config ( Probably calls some script / make target ) and we can do the same. If it's something internal to bazel ( though I don't see how ), then we should definately continue using that one.

Reply:
Oh wait, the image is already bassed on build_kubetest and that has bazel. Nevermind, we can use bazel then.